### DISCUSSION TITLE
Bootstrapper license

### DISCUSSION THREAD

**Discussion by Cendor** [MicrosoftEdge/WebView2Feedback#3079] [Q&A]
Hello everyone,

we develop an application which uses WebView2.
Now, we want to distribute WebView2 in the Evergreen mode via bootstrapper, which shall be included in our installer.

Where can I find the license agreement for the WebView2 bootstrapper? So far I have only been able to find the license agreement for the WebView2 Runtime.

Best regards,
Cender

**Comment by studyliuup** [comment]
你好，你的邮件我已收到，我会尽快给你回复。谢谢。
